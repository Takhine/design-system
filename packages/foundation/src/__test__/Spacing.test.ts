import Spacing from '../Spacing';

test('snapshop of fontsizes', ()=> {
    expect(Spacing).toMatchSnapshot()
})