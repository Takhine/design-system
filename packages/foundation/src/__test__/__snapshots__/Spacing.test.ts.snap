// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshop of fontsizes 1`] = `
{
  "lg": "lg",
  "md": "md",
  "none": "none",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;
